lib/travis/build/script/r.rb

```diff
@@ -482,7 +487,7 @@ def setup_pandoc
         # Install gfortran libraries the precompiled binaries are linked to
         def setup_fortran_osx
           return unless (config[:os] == 'osx')
-          if r_version < '3.4'
+          if r_version_less_then('3.4')
```

Comment:
I'm sorry to pick nits, but this should be `r_version_less_than`. Otherwise it looks good. :-)